src/Runtime/LambdaRuntime.php

```diff
@@ -243,6 +243,12 @@ public function failInitialization(string $message, ?\Throwable $error = null):
      */
     private function postJson(string $url, $data): void
     {
+        $contentType = $data['multiValueHeaders']['content-type'][0] ?? null;
+        if ($contentType && substr($contentType, 0, 4) !== 'text') {
```

Comment:
would `application/json` also be base64 encoded then?

Reply:
Right it seems like this would match a lot of non-binary responses.

Reply:
Actually it sounds reasonable to me after thinking about it (see https://github.com/brefphp/bref/pull/457#issuecomment-533119294).

:+1: `application/json` should be added here.